symbology/read-symbols-from-mobile-style-file/src/main/java/com/esri/samples/read_symbols_from_mobile_style_file/ReadSymbolsFromMobileStyleFileController.java

```diff
@@ -173,33 +159,33 @@ private void buildCompositeSymbol() {
     List<String> symbolKeys = Arrays.asList("Face1", eyesKey, mouthKey, hatKey);
 
     // get the symbol from the SymbolStyle
-    ListenableFuture<Symbol> symbolFuture = emojiStyle.getSymbolAsync(symbolKeys);
-    symbolFuture.addDoneListener(() -> {
-      try {
-        faceSymbol = (MultilayerPointSymbol) symbolFuture.get();
-        if (faceSymbol == null) {
-          return;
+    emojiStyle.getSymbolAsync(symbolKeys).toCompletableFuture().whenComplete(
+      (symbol, ex)  -> {
```

Comment:
I would split the lines with the `whenComplete(` on the same line as `(symbol, ex)  -> {` e.g.
```
.whenComplete((symbol, ex)  -> {
```